src/tests/ftest/io/mdtest_small.yaml

```diff
@@ -38,7 +42,7 @@ mdtest:
     branch: !mux
         zero:
             branching_factor: 1
-            num_of_files_dirs: 1000
+            num_of_files_dirs: 1
```

Comment:
why did you change this to 1?
im not sure if im missing something in this yaml file, but that just creates a single empty file / dir from mdtest perspective, which really is close to not doing anything..

Reply:
Was accidentally pushed as '1' made it back to 1000.